project/hackyourweather-4/src/pages/Home.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import AppTitle from '../components/AppTitle';
+import CardList from '../components/CardList';
+import Form from '../components/Form';
+
+function App() {
+  const [status, setStatus] = useState('loading');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [cities, setCities] = useState([]);
+
+  const fetchWeather = async city => {
+    try {
+      const res = await fetch(
+        `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&APPID=${process.env.REACT_APP_OPENWEATHERMAP_API_KEY}`,
+      );
+      if (!res.ok) {
+        throw new Error('Oops! The city name is not correct. Please try again');
+      }
+      const newCity = await res.json();
+
+      const addCity =
+        cities.filter(city => city.id === newCity.id).length === 0 || cities.length === 0;
```

Comment:
Maybe this could win the simplicity reward:
```suggestion
        !cities.some(city => city.id === newCity.id) || cities.length === 0;
```

Reply:
Thanks for the elegant tip. 
I never used .some() method before.